.npmignore

```diff
@@ -0,0 +1,8 @@
+.github/
+assets/
```

Comment:
probably, image wont be shown in NPM

Reply:
no. this file cleans up the package to be published and available for installation.

npm, it turns out, takes the path to the image from the repository, which can be specified in package.json